bash_completion

```diff
@@ -2432,6 +2457,28 @@ _xfunc()
     "$@"
 }
 
+_comp_deprecate_func _apt_cache_packages _comp_xfunc_apt_cache_packages
+_comp_deprecate_func _apt_cache_sources _comp_xfunc_apt_cache_sources
+_comp_deprecate_func _apt_cache_src_packages _comp_xfunc_apt_cache_src_packages
+_comp_deprecate_func _comp_cmd_apt_get_installed_packages _comp_xfunc_apt_get_installed_packages
+_comp_deprecate_func _asciidoc_doctype _comp_xfunc_asciidoc_doctype
+_comp_deprecate_func _cvs_roots _comp_xfunc_cvs_roots
+_comp_deprecate_func _comp_dpkg_installed_packages _comp_xfunc_dpkg_installed_packages
+_comp_deprecate_func _iconv_charsets _comp_xfunc_iconv_charsets
+_comp_deprecate_func _mailman_lists _comp_xfunc_list_lists_mailman_lists
+_comp_deprecate_func _mysql_character_sets _comp_xfunc_mysql_character_sets
+_comp_deprecate_func _python_modules _comp_xfunc_python_modules
+_comp_deprecate_func _python_warning_actions _comp_xfunc_python_warning_actions
+_comp_deprecate_func _rpm_installed_packages _comp_xfunc_rpm_installed_packages
+_comp_deprecate_func _scp_local_files _comp_xfunc_ssh_scp_local_files
+_comp_deprecate_func _scp_remote_files _comp_xfunc_ssh_scp_remote_files
+_comp_deprecate_func _ssh_identityfile _comp_xfunc_ssh_identityfile
+_comp_deprecate_func _ssh_options _comp_xfunc_ssh_options
+_comp_deprecate_func _ssh_query _comp_xfunc_ssh_query
+_comp_deprecate_func _ssh_suboption_check _comp_xfunc_ssh_suboption_check
```

Comment:
@scop has suggested in https://github.com/scop/bash-completion/issues/732#issuecomment-1094831476 that the backward-compatibility names might be defined in a separated file (e.g. in `/etc/bash_completion.d`), but currently these compatibility names are tentatively included in `bash_completion`.
- If we put the file in `/etc/bash_completion.d`, something unwanted might occur when the user sets the custom `BASH_COMPLETION_COMPAT_DIR`, so it might be better to be in another directory such as `helpers`?
- Another option is to put these compatibility names in the same file as the new API. For example, define the compatibility name in _apt_cache_packages.

Reply:
I now noticed that this causes the problem when the new name is not yet defined when the compatibility name is used. So, the compatibility name should be defined after the new name is defined (for example in the same file as the new name), or the function of the compatibility name should call the function of the new name through `_comp_xfunc`.